datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java

```diff
@@ -91,6 +93,7 @@ protected SdxContext createFlowContext(FlowParameters flowParameters, StateConte
             protected void prepareExecution(DatalakeTriggerBackupEvent payload, Map<Object, Object> variables) {
                 variables.put(OPERATION_ID, payload.getDrStatus().getOperationId());
                 variables.put(REASON, payload.getReason().name());
+                variables.put(MAX_DURATION_IN_MIN, payload.getMaxDurationInMin());
```

Comment:
Treat duration in DatalakeTriggerBackupEvent and DatalakeDatabaseBackupStartEvent differently.

Duration in DatalakeDatabaseBackupStartEvent should be used to control the backup duration for database backup operation.

Duration in DatalakeTriggerBackupEvent should be used to configure the overall backup duration. It should be used to control durationInMinutes in DatalakeFullBackupWaitHandler.

Please make similar changes for restore as well.

Reply:
Thanks @kkalvagadda1 ! Want to make sure I understand this - does it mean when we are calling the API, we should provide two inputs? One is DB duration, and another one is the overall backup duration?

Reply:
Thanks @kkalvagadda1 ! Want to make sure I understand this - does it mean when we are calling the API, we should provide two inputs? One is DB duration, and another one is the overall backup duration?

Reply:
Hi @kkalvagadda1 , updated this so DatalakeTriggerBackupEvent doesn't have that DB-related value (it was actually in its parent class `DatalakeDatabaseDrStartBaseEvent ` but I removed it from that). Please let me know if the update looks good. Thank you! Now `DatalakeTriggerBackupEvent` only has `fullDrMaxDurationInMin ` and `DatalakeDatabaseBackupStartEvent ` only has `databaseMaxDurationInMin `.